### ISSUE TITLE
domain of exists_at not well aligned with its Elucidation in bfo-2020-terms

### ISSUE THREAD

**Issue by psiotwo** [BFO-ontology/BFO-2020#41]
In `bfo-2020-terms.xlsx` (BTW, pity not to see it in some non-proprietary format like CSV), on line 5 (exists_at):

While Elucidation speaks about "relation between a **particular** and ..", the domain is **entity**. 

This is confusing.

**Comment by alanruttenberg** [comment]
Thank you for pointing this out. The top of the hierarchy really should be called particular. That it isn't is a legacy. It is further complicated by the fact that that Barry defines entity as a particular OR a universal. A reasonable thing would be to add "particular" as an alternative term, or as the sole term. Unfortunately, there isn't yet a reference for BFO-2020, only what's in ISO 21838. I'll leave this open for when we address documentation.

**Comment by phismith** [comment]
I think Entity is fine as it stands
We just do not take advantage of the fact that Universal falls under Entity. (But someone could in the future.)
For the mathematics ontology we are working on will need structures, such as numbers, geometrical shapes, and many more, which are entities, but neither universals nor particulars.

**Comment by alanruttenberg** [comment]
The problem is that in OWL the domain of discourse doesn't include classes. In the FOL it does. So the OWL assertion is saying, effectively, that no entity is a class. The distinction between universals and class doesn't matter here - what matters are that the sorts of things that have extensions, which both classes and universals have, are not members of the class owl:Thing. Whereas by saying entity is a subclass of owl:Thing, we are are saying they are.

**Comment by phismith** [comment]
Yet another piece of evidence that OWL is infected by a reductionist philosophy which denies the existence of universals. (In fact doesn't even allow us to formulate the thesis that universals exist.)
But I see your point. If we change 'entity' to 'particular' in the OWL version we should not engage in comparable self-harm in the FOL version.

**Comment by alanruttenberg** [comment]
Yet another example of the assertion that OWL has a philosophy.  OWL no more has a philosophy than FOL does. You may take exception with my *use* of OWL. I *could* represent things with extensions in OWL if I was content with OWL's semantics not making useful inferences related to extensions so represented. But that would make OWL fairly useless for inference. So instead, I use OWL to the extent possible. That use means that entity, if the union of particular and universal, can not be represented using the strategy I've used. The strategy does fine if instead of entity the top of the hierarchy *in OWL* particular is used. In that case, as you put it, we "just do not take advantage of the fact that Universal falls under Entity". Someone else can, as long as that that someone uses a more powerful fragment of FOL than OWL, or unless that someone can think of a more clever use of OWL than I have been able to come up with.

**Comment by alanruttenberg** [comment]
As you point out, we would do this in OWL and not in FOL, since we're doing fine with entity in FOL.

**Comment by alanruttenberg** [comment]
The most likely strategy for using OWL in the case that there is a new category that is disjoint from entity and particular will be to have the new category be a disjoint sibling of entity underneath owl:Thing. We could do a similar thing with universal - make it sibling to particular in OWL. The problem with doing that would be that it would lead to confusion. There would be an owl class that is the closest approximation to a universal, as is the case now, as well as an owl individual for the same universal, and assertions on one would not affect assertions on the other.

Having a category disjoint from entity will cause other damage. The ISO definition for entity 

>  ‘Entity’ is defined in this standard following the ISO 1087-1:2003 definition of ‘object’ as ‘anything perceivable or conceivable’. This allows the term ‘entity’ to be used also to represent for example what is planned or postulated.

I'm assuming that members of this other category are conceivable, otherwise we wouldn't be talking about them.

**Comment by alanruttenberg** [comment]
My previous comment has been deleted as it was based on my misread of the new category being disjoint from entity. You said these were also entities.

**Comment by alanruttenberg** [comment]
After discussion with Barry the plan will be to, after soliciting comments from bfo-discuss and barring objections, change the label of "entity" to "particular".

**Comment by psiotwo** [comment]
Makes good sense to me. 

Might also be worth to equip http://purl.obolibrary.org/obo/BFO_0000001 with some editorial note summarising the rationale behind this naming change for (BFO 2 -> BFO 2020)ers.

**Comment by johnbeve** [comment]
@alanruttenberg I assume this is OBE since there was a decision to change the label of "entity" to "particular" but nothing came of it?

**Comment by alanruttenberg** [comment]
No, it just hasn't been implemented, and the reason it hasn't been implemented is because I'd like there to be a an orderly way in which this is rolled out and I have been hesitant to do this on my own. Now that you are involved maybe we can talk about the best way to announce this before making the change.
I would keep entity as an alternative term.

**Comment by wdduncan** [comment]
FWIW, I think changing `entity` to `particular` is not a good a idea. I don't it really buys you anything, and the reason for doing so seem to resolve around the philosophical/metaphysical notion of universals being particulars. Many in the OBO Foundry will "roll their eyes" at this (myself included). `Entity` has existed as a term for long time and changing it not worth it. 

For myself, the only reason I like having `entity` is so that I can make a sibling `obsolete class`. If it wasn't for that, I would fine putting the classes under `owl:Thing`. I live in OWL land (sorry not sorry). But, this is where most applied ontologist live, and I'm interested in having debates about this. 

If this issue has already been decided, then my opposition is a moot point. Maybe it will go off without a hitch.

**Comment by phismith** [comment]
I agree with Bill - we should keep 'entity'; perhaps improve its elucidation
BS

**Comment by alanruttenberg** [comment]
@phismith We already discussed this and you already agreed to make the change. Please review this discussion on the Github site, particularly https://github.com/BFO-ontology/BFO-2020/issues/41#issuecomment-1249698230
This is a change to the OWL version, not to the FOL.

**Comment by johnbeve** [comment]
Are users of the OWL version confused by the rdfs:label being 'entity' rather than 'particular'? I suspect not. 
Will users of the OWL version be confused by shifting the rdfs:label to 'particular'? Given how long 'entity' has been in use (in the OWL versions), I suspect yes. 

I'm in favor of following an earlier @alanruttenberg's suggestion (https://github.com/BFO-ontology/BFO-2020/issues/41#issuecomment-1248607201) to add 'particular' as an altlabel

**Comment by wceusters** [comment]
> Thank you for pointing this out. The top of the hierarchy really should be called particular. That it isn't is a legacy. It is further complicated by the fact that that **Barry defines entity as a particular OR a universal.** A reasonable thing would be to add "particular" as an alternative term, or as the sole term. Unfortunately, there isn't yet a reference for BFO-2020, only what's in ISO 21838. I'll leave this open for when we address documentation.

and

> I think Entity is fine as it stands
We just do not take advantage of the fact that Universal falls under Entity. (But someone could in the future.)
For the mathematics ontology we are working on will need structures, such as numbers, geometrical shapes, and many more, which are entities, but neither universals nor particulars.

-----------

In the CLIF files, only occurrents and continuants are qualified as 'entity' (vgn-1).
I would support an extra axiom (forall (u) (if (universal u) (entity u))) sticking to the current way of representing.

But if 'entity' is to be expanded as Barry indicates - and I would add 'portion of reality' to the list - then it is better to represent them as individuals in the domain, and not as unary relations. Although FOL doesn't have a philosophy, I prefer representations that stick closer to the philosophy of BFO.

**Comment by alanruttenberg** [comment]
The relata of exists-at are particulars

  (cl:comment "Relata of exists-at are particulars. [oap-1]"
    (forall (i t)
     (if (exists-at i t)
      (and (particular i) (particular t)
       (instance-of t temporal-region t)))))

There has never been the suggestion that universals exist in time the same way that particulars do.

**Comment by alanruttenberg** [comment]
There's also 
```
(cl:comment "If you exist you instatiate a universal and vice-versa [bee-1]"
    (forall (a t)
     (iff
      (exists (u)
       (and (universal u) (instance-of a u t)
        (instance-of t temporal-region t)))
      (and (particular a) (instance-of t temporal-region t)
       (exists-at a t)))))
```
[ugh type needs to be fixed]

That was written carefully to not imply that entity was in the domain of exists-at.

There's an axiom in the source, not in the release probably due to an omission, that entity is the union of universals and particulars, as I believe that's what the ISO spec says.

The reason the domain of exists-at wasn't in the OWL was because it couldn't be proved, because it isn't true according to the theory. 

One of the reasons to change "entity" to "particular" is to be able to remedy this.

**Comment by alanruttenberg** [comment]
I will correct myself.  The BFO2 reference has, in a section "Relations defined for any entity"

```
2.12.3	The exists_at relation
ELUCIDATION: b exists_at t means: b is an entity which exists at some temporal region t. [118-002]
DOMAIN: entity
RANGE: temporal region
```
However, I'm not sure that really intended to mean that a universal can be the subject of exists_at and the axiom isn't definitive since it's not an iff. The section name is clearly not definitive since it includes instance_of in the list, although it then qualifies it to say (I'm paraphrasing) well not really, it relates particulars and universals.

I'm going to check but I seem to remember that the inclusion of universals in entity was a late addition to BFO.

**Comment by alanruttenberg** [comment]
Ok, more archeology. The BFO2 reference, in the editing version, has, in the section "1.2	Summary of most important changes in BFO 2.0 as compared to BFO 1.1"
The note: "New relation exists_at added."

We'll see what @phismith says but I think implying the domain of exists_at including universals was a mistake. For instance, is-a is not temporally indexed as instance-of is.

"2.5 Avoiding is_a overloading
In ordinary English the following assertions are equally grammatical: 
(a) a human being is a mammal
(b) a professor is a human being
(c) John is a human being
(d) a restaurant in Palo Alto is a restaurant
However, the meaning of ‘is a’ is quite different in each case, and ontologies which do not take account of these differences are guilty of what Guarino has called “‘is a’ overloading” [80]. Here only (a) and (b) are properly to be treated in terms of the is_a relation between universals or types. 
"

**Comment by alanruttenberg** [comment]
I checked with Barry re: Domain of *exists_at*. I confirmed that it wasn't the original intention to have the domain include *universal*, but he says he's not against it.

I'd rather not change it from the current right now - perhaps in a future release of BFO. First of all it, it would require changes to the axiomatization and I don't have time to work through those at the moment. Second, it would imply, I think, other facts about *universal* that I don't think we've actually thought through. 

So my recommendation is still to change the top term (in the OWL) to be *particular*, and then have the domain of *exists_at* be *particular*. If at some future point we elaborate on *universal*s or want other categories below *entity* we can add *entity* above *particular*, and the new categories sibling to *particular*.

Two other potential alternatives:
1. We can interpose *particular* between *entity* and *continuant/occurrent* now, and have the domain of *exists_at* be *particular*.
2. We can keep *entity* and make the domain of *exists_at* be the union of *continuant* and cccurrent.
